src/js/editor/event-manager.js

```diff
@@ -45,6 +45,14 @@ export default class EventManager {
     this._textInputHandler.register(inputHandler);
   }
 
+  unregisterInputHandler(name) {
+    this._textInputHandler.unregister(name);
+  }
+
+  unregisterAllTextInputHandlers() {
+    this._textInputHandler.destroy();
```

Comment:
yes, `destroy()` in textInputHandler actually is not destroying the `textInputHandler`, it's assigning an empty array to textInputHandler. 

But for clearer purpose, I will assign a new textInputHandler after destroying.